idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -1518,7 +1652,7 @@ def initialize(blk, gas_phase_state_args=None, solid_phase_state_args=None,
                         property package(s) to provide an initial state for
```

Comment:
Moving towards IDAES 2.0, we are renaming this method to `initialize_build`. There is a general purpose `initialize` method in the parent class which will automatically call `initialize_build`, so nothing changes for the end user. However, this allows us to automate handling of things that get added on to the unit model (such as costing blocks).

Reply:
Are there any potential compatibility issues with renaming this now? I know @chineduobiora would like to reference these IDAES models in external flowsheets, so although tedious it may be better to open a future PR to update all unit models, flowsheets and tests together.

Reply:
@chineduobiora I can make the changes Andrew suggests in the MB model now since we have an open PR to push commits to. For the FB0D model, I believe it would be best to open a new PR at some point to add these updates along with scaling/tests for consistency with the other CLC models.

Reply:
@bpaul4 sounds good